bubble/bubble/Models/Bubble.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Bubble.swift
+//  bubble
+//
+//  Created by Siraj Zaneer on 2/8/18.
+//  Copyright © 2018 CS 408. All rights reserved.
+//
+
+import UIKit
+import Firebase
+
+class Bubble {
+
+    var text: String!
+    var owner: String!
+    var voteCount: Int!
+    var uid: String!
+    var geopoint: GeoPoint!
+    
+    init(bubbleData: [String: Any]) {
+        text = bubbleData["text"] as! String
+        owner = bubbleData["owner"] as! String
+        voteCount = bubbleData["voteCount"] as! String
+        uid = bubbleData["voteCount"] as! String
+        geopoint = bubbleData["location"] as! GeoPoint
```

Comment:
Can you downcast instead of force unwrap the variables from bubbleData here?

Reply:
So for each Bubble these variables are required so they should exist, otherwise the app will crash in a different part of the application so doing it here will allow us to find issues with storage/retrieval of bubbles quicker.

Reply:
damn didn't know you could do that lol nice